x/cork/types/params.go

```diff
@@ -53,7 +53,7 @@ func validateVoteThreshold(i interface{}) error {
 		return errors.New("vote threshold cannot be nil")
 	}
 
-	if voteThreshold.LTE(sdk.ZeroDec()) || voteThreshold.GT(sdk.OneDec()) {
+	if voteThreshold.LT(sdk.ZeroDec()) || voteThreshold.GT(sdk.OneDec()) {
```

Comment:
This is the only open question I really have with this and am curious as to the answer. I assume it's related to the test maybe, but I agree I can't think of a scenario where we'd want this to be zero. We might even want to set a minimum value for this that it can't be below.

Reply:
The log message for the param validation implied it expected to allow 0, and the difference was breaking tests. Open to lower bound suggestions.